src/routes/index.js

```diff
@@ -2,16 +2,26 @@ const express = require('express');
 const router = express.Router();
 const Products = require('../models/Products');
 
-router.get('/', async (req,res) => {
+router.get('/', async (req, res) => {
     try {
-        const prds = await Products.find();
+        const products = await Products.find();
+        if (!products) return res.status(404).render("pages/error", {
+            title: "Moramia <3",
+            error: 404,
+            refreshUrl: process.env.BROWSER_REFRESH_URL, //browser refresh listening
+        });
         res.render("pages/index", {
             title: "Moramia <3",
-            products : prds,
+            products: products,
+            refreshUrl: process.env.BROWSER_REFRESH_URL, //browser refresh listening
+        });
+    } catch (err) {
+        console.log(err);
+        res.status(500).render("pages/error", {
+            title: "Moramia <3",
```

Comment:
same for 500 title: Try something like: `Internal server error`

Reply:
Done